heron/common/tests/cpp/network/order_unittest.cpp

```diff
@@ -68,16 +68,20 @@ class Terminate : public Client {
 
 static OrderServer* server_;
 
-void start_server(sp_uint32 port) {
+void start_server(sp_uint32* port, CountDownLatch* latch) {
```

Comment:
@huijunw - instead of * can you use reference like sp_uint32& port?  It is much more easy to read.